pwiz_tools/Skyline/Controls/Graphs/SpectrumGraphItem.cs

```diff
@@ -269,7 +269,9 @@ public override PointAnnotation AnnotatePoint(PointPair point)
                 case IonType.b: fontSpec = FONT_SPEC_B; break;
                 case IonType.y: fontSpec = FONT_SPEC_Y; break;
                 case IonType.c: fontSpec = FONT_SPEC_C; break;
-                case IonType.z: fontSpec = FONT_SPEC_Z; break;
+                case IonType.z:
+                case IonType.zh:
+                case IonType.zhh: fontSpec = FONT_SPEC_Z; break;
```

Comment:
Since I introduced specific colors for zh and zhh ions I guess I should add the font specs as well.